### ISSUE TITLE
PTXprint doesn't print with 1.1.9 in a fresh Wasta 20.04 installation

### ISSUE THREAD

**Issue by KimB2017** [sillsdev/ptx2pdf#221]
Is someone else testing in Wasta 20.04? I just installed ptxprint 1.1.9 from the PPA in a new system (all pkgs updated). PT9 is installed and I can edit the project files so it's not a permission thing on that end.

PTXprint opens and shows correct settings for a few saved configurations. I can generate a piclist and edit the positions, etc, and save it. I've seen a problem similar to that noted in #219, the piclist will disappear until I close and reopen ptxprint, although if I click "regenerate" it says the list is present.

But when I click Print nothing happens--no temp files, not even a log file is created in PrintDraft folder. (yes, that checkbox is ticked). I've tried turning off pics and all the config files in Advanced with no success.

The same saved configurations print with no problems on a Wasta 18 computer in v 1.1.8 and 1.1.9.
It looks as if all the dependencies are installed correctly, including python 3.8.2.
If it's working for others in 20.04, what can I do to try to debug it?

**Comment by mhosken** [comment]
Can you check to see if you have texlive-xetex installed (`apt search xetex`). On Linux we don't use xetex from Paratext, we use the xetex installed on the system. Also, if you run ptxprint from a terminal, you should get some informative messages as to why the Print button is unhappy.

**Comment by KimB2017** [comment]
Yes, xetexlive-xetex is installed. Updated to 1.1.9.6--different error appears now, and it "sometimes" prints.

Terminal error message is very consistent when it fails. I'm still trying to pin down what settings cause failure:
`  File "/usr/lib/python3/dist-packages/ptxprint/runjob.py", line 404, in sharedjob
    texf.write(info.asTex(filedir=self.tmpdir, jobname=outfname.replace(".tex", ""), extra=extra))
  File "/usr/lib/python3/dist-packages/ptxprint/texmodel.py", line 716, in asTex
    res.append(fn(v, self))
  File "/usr/lib/python3/dist-packages/ptxprint/snippets.py", line 353, in generateTex
    cpystr = re.sub('_', '\u00A0', artstr[artistWithMost][0])
KeyError: 'Global Rec'

It always fails with the quoted bit in the last line being one of the abbreviations I have placed in the "copyright" field of the \fig spec. (I haven't gotten past this error yet, to update this field to the full copyright statement and test the new copyright insertion routine--which is an amazing addition to ptxprint, by the way!)
`
